src/main/java/wolox/training/models/User.java

```diff
@@ -49,6 +57,8 @@
   @ApiModelProperty(notes= "A book can be in more than one user book collection.")
   @JsonIgnoreProperties(value = "users")
   private List<Book> books = new ArrayList<>();
+  @Column(name="user_type", insertable = false, updatable = false)
+  private String user_type;
```

Comment:
We follow the convention of naming the attributes in camel case, so it should be `userType`. If you want it to be shown with the name "user_type" you should use the annotation `@JsonProperty`

Reply:
Right! I'll fix it